pinot-common/src/main/java/org/apache/pinot/common/function/scalar/StringFunctions.java

```diff
@@ -389,4 +389,15 @@ public static int hammingDistance(String input1, String input2) {
   public static boolean contains(String input, String substring) {
     return input.contains(substring);
   }
+
+  /**
+   * Compare input strings lexicographically.
+   * @return the value 0 if the first string argument is equal to second string; a value less than 0 if first string
+   * argument is lexicographically less than the second string argument; and a value greater than 0 if the first string
+   * argument is lexicographically greater than the second string argument.
+   */
+  @ScalarFunction
+  public static int compare(String input1, String input2) {
```

Comment:
Let's rename it to `strcmp` which is a standard MySQL function. Since we don't have type matching yet, we want to differentiate the string comparison from comparison of other types

Reply:
Renamed.

Reply:
Renamed.